deploy/role.yaml

```diff
@@ -360,4 +361,11 @@ rules:
       - watch
       - update
       - create
-  # remove to here
\ No newline at end of file
+  # remove to here
+  # Needed for granting fuse only view
+  - apiGroups:
+    - build.openshift.io
+    resources:
+    - jenkins
+    verbs:
+    - "*"
```

Comment:
can we put an end comment too please. i.e. `# end permissions for granting fuse only view`